screenpipe-vision/Cargo.toml

```diff
@@ -96,13 +96,14 @@ windows = { version = "0.58", features = [
   "Storage",
   "Storage_Streams",
 ] }
-xcap = "0.0.12"
+xcap = "0.3.1"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 libc = "=0.2.164"
 xcap-macos = { package = "xcap", git = "https://github.com/mediar-ai/xcap", rev = "965bc99" }
+xcap = "0.3.1"
```

Comment:
hmm forgot something 

i am not sure the focus impl in latest `xcap` is want we want or how does it work with our current focus-related code

https://github.com/nashaofu/xcap/issues?q=focus

any chance you can test out how we capture windows with this version of xcap, like which condition, is it when mouse in window, etc., what happen when multi monitor in terms of focus?

Reply:
can you give a few instructions on how to test this?

alternatively, you can pull the latest changes from xcap, and we can use the current capture impl from mediar-ai/xcap with the new labels thing